scripts/api-inference/templates/task/image-text-to-text.handlebars

```diff
@@ -0,0 +1,30 @@
+## Image-Text to Text
+
+Image-text-to-text models take in an image and text prompt and output text. These models are also called vision-language models, or VLMs. The difference from image-to-text models is that these models take an additional text input, not restricting the model to certain use cases like image captioning, and may also be trained to accept a conversation as input.
+
+{{{tips.linksToTaskPage.image-text-to-text}}}
+
+### Recommended models
+
+{{#each models.image-text-to-text}}
+- [{{this.id}}](https://huggingface.co/{{this.id}}): {{this.description}}
+{{/each}}
+
+{{{tips.listModelsLink.image-text-to-text}}}
+
+### Using the API
+
+{{{snippets.image-text-to-text}}}
+
+### API specification
+
+#### Request
+
+{{{specs.image-text-to-text.input}}}
```

Comment:
This doesn't seem to be populated by `generate.ts`. This is because `image-text-to-text` doesn't have specs in huggingface.js: https://github.com/huggingface/huggingface.js/tree/main/packages/tasks/src/tasks/image-text-to-text.

In fact we will want to reuse the specs from the chat-completion specs which already describes how to use it with images (see [here](https://github.com/huggingface/huggingface.js/blob/a22a62a0cfc9fa644a9d40e4bbfcd0489cc11038/packages/tasks/src/tasks/chat-completion/spec/input.json#L201)).

What I would suggest is to keep a small `image-text-to-text.handlebars` page with task description, recommended models, inference snippets and then in the `### API specification`, redirect to "chat completion" page saying that it's the same API.

And on the chat completion page maybe add a sentence to tell that it works both with conversational LLMs and conversational VLMs.

Reply:
We can add duplicate inference snippets + recommended models for VLMs in both the "chat completion" page and in the "image text to text page". It would be duplicated information but at least we are sure users won't miss it (+ it's autogenerated so easy to maintain).

This situation is a bit the side effect of the fact that `Chat Completion` is not a proper task by itself. But I still think it's worth having it for SEO / discoverability.

Reply:
+1, Fixed!